pkg/depsdev/definitions/project.go

```diff
@@ -18,9 +18,9 @@ import "time"
 
 type Project struct {
 	ProjectKey      ProjectKey `json:"projectKey,omitempty"`
-	OpenIssuesCount string     `json:"openIssuesCount,omitempty"`
-	StarsCount      string     `json:"starsCount,omitempty"`
-	ForksCount      string     `json:"forksCount,omitempty"`
+	OpenIssuesCount int        `json:"openIssuesCount,omitempty"`
```

Comment:
It would be better to change all the occurences of this issue. For example, https://github.com/edoardottt/depsdev/blob/d19b20cf6a023083cad2a32e06e4d4dbfb3e183a/pkg/depsdev/project.go#L53 should be a number as far as I can see in the docs.  
What do you think?

Reply:
references: https://docs.deps.dev/api/v3/

Reply:
I've added some other type fixe I could find. Any other I've missed?